### ISSUE TITLE
changes in conf.py do not trigger a new build

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#179]
After conf.py is changed, 

```
make html
```

should trigger a full build of the documentation.

Right now it just says

```
loading pickled environment... done
building [html]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
no targets are out of date.
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/179
- Originally reported by: [rsb](https://bitbucket.org/rsb)
- Originally created at: 2009-05-21T13:46:42.922

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-05-22 16:16:51+00:00_

What exactly did you change in conf.py? The options are put into different groups depending on what influence they have on a needed rebuild.

**Comment by shimizukawa** [comment]
_From [rsb](https://bitbucket.org/rsb) on 2009-05-25 08:45:00+00:00_

I thought there were more, but I can only reproduce it for the one:

//pngmath_latex_preamble//

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-05-27 19:59:38+00:00_

Indeed the pngmath config values' rebuild parameter was not set properly. Fixed in <<cset 8e574c4a10b1013e8308c7c175b4b05503e894ea>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:08+00:00_

Removing version: 0.6 (automated comment)
